### PR TITLE
CAMEL-15371: underline links

### PR DESCRIPTION
**Pull request by AemieJ**

### CODE REVIEW

File: antora-ui-camel/src/css/doc.css [hunk 1]
```diff
@@ -149,6 +149,16 @@
 .doc a {
   color: var(--link-font-color);
   word-break: break-word;
+  text-decoration: underline;
+  text-underline-position: under;
```
Comment (zregvart) [inline antora-ui-camel/src/css/doc.css:153]: This bit looks a bit weird to me.

```suggestion
```

Comment (AemieJ) [reply antora-ui-camel/src/css/doc.css:153]: You require this particular line so that there is a minimum distance between the underline and the text.

Comment (zregvart) [reply antora-ui-camel/src/css/doc.css:153]: Right, so this is with `text-underline-position: under`:

Screenshot_2020-08-11 Home - Apache Camel

and without:

Screenshot_2020-08-11 Home - Apache Camel(1)

I think what you're reacting to is that the underline is quite noticeable, perhaps adding to make it lighter:

```suggestion
  text-decoration-thickness: 1px;
  text-decoration-style: dotted;
```

Would help, this is how it renders in Firefox:

Screenshot_2020-08-11 Home - Apache Camel(3)

Comment (AemieJ) [reply antora-ui-camel/src/css/doc.css:153]: I am not sure about to dotted style. It would look quite odd. I will look into the thickness.

Comment (AemieJ) [reply antora-ui-camel/src/css/doc.css:153]: Thickness feature isn't present for chrome browser and the dotted style looks inconsistent. Either we just not include under or we include it. It's best to keep it simple. Or I could use the `border-bottom` property.

**Comment by zregvart** [comment]
Not sure about the underline being below the descenders. My personal preference would be

```css
text-decoration: underline;
text-decoration-style: dotted;
```

There are some methods on different underline styles at [css tricks](https://css-tricks.com/styling-underlines-web/) and I remember reading about the underline journey at [medium](https://medium.design/crafting-link-underlines-on-medium-7c03a9274f9) (for inspiration).

@aashnajena and @delawen any thoughts on this?

**Comment by AemieJ** [comment]
@zregvart alright I got your concern about it but with those changes, it looks messy in chrome browser, I found a method using background-image and linear-gradient on the CSS tricks site. 

```
background-image: linear-gradient(to right, #583ac2 50%, transparent 50%);
background-position: 0 1.1em;
background-repeat: repeat-x;
background-size: 7px 1px;
```

If it's okay with you, I will implement it in this manner, it will create a neater effect of underline on each browser. 
underline-link

**Comment by zregvart** [comment]
> ```
> background-image: linear-gradient(to right, #583ac2 50%, transparent 50%);
> background-position: 0 1.1em;
> background-repeat: repeat-x;
> background-size: 7px 1px;
> ```

Does the `background-position` track different font sizes correctly? Might be a bit of a headache to have this properly positioned for all font sizes.

**Comment by AemieJ** [comment]
@zregvart I have checked it, the `background-position` tracks all the font-size of the text and that of browsers correctly.

**Comment by zregvart** [comment]
Thanks @AemieJ

**Pull request merged**